Source/Csla.AspNetCore/Blazor/ApplicationContextManagerInMemory.cs

```diff
@@ -188,10 +188,8 @@ public void SetLocalContext(IContextDictionary localContext)
     /// Gets the client context.
     /// </summary>
     /// <param name="executionLocation"></param>
-    public IContextDictionary GetClientContext(ApplicationContext.ExecutionLocations executionLocation)
+    public IContextDictionary? GetClientContext(ApplicationContext.ExecutionLocations executionLocation)
     {
```

Comment:
The getter should create an instance on demand so the end developer never gets a null, even if the private value was null behind the scenes.

Reply:
Same as with the LocalContext. Missed reverting that.